syntax = "proto3";

message LayerOutput {
    uint32 index = 1;
    string name = 2;
    string op_type = 3;
    repeated uint32 dims = 4;
    repeated float value = 5;
}

message ModelOutput {
    repeated LayerOutput layer_out = 1;
}
